src/context.rs

```diff
@@ -15,6 +16,19 @@ where
     mb: Option<Mailbox<A>>,
 }
 
+impl<A: Actor<Context = Context<A>>> fmt::Debug for Context<A> {
+    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+        let mb = match self.mb {
+            Some(_) => "Some(_)",
+            None => "None",
+        };
+        fmt.debug_struct("Context")
+            .field("parts", &self.parts)
+            .field("mb", &format!("{}", mb))
```

Comment:
I think `format` is not needed here?

Also shouldn't Mailbox already impl Debug?

Reply:
I have not found Debug implementation for Mailbox.

Reply:
Oh true, but since it is public API, should we impl Debug for it too?

Reply:
I don't know how to impl Debug for MB. It has just one parametrized field...'